app/components/search-facet-registration-type.js

```diff
@@ -43,16 +48,22 @@ export default Component.extend({
     }),
     init() {
         this._super(...arguments);
-        this.registrationTypes = [
-            'AsPredicted Preregistration',
-            'Election Research Preacceptance Competition',
-            'Open-Ended Registration',
-            'OSF-Standard Pre-Data Collection Registration',
-            'Prereg Challenge',
-            'Pre-Registration in Social Psychology (van \'t Veer & Giner-Sorolla, 2016): Pre-Registration',
-            'Replication Recipe (Brandt et al., 2013): Post-Completion',
-            'Replication Recipe (Brandt et al., 2013): Pre-Registration',
-        ];
+        this.get('store')
+            .findAll('metaschema')
+            .then(this._returnResults.bind(this))
+            .catch(this._errorMessage.bind(this));
+    },
+    _returnResults(results) {
+        const typeArr = [];
+        results.forEach(function(result) {
```

Comment:
I think I prefer a map here if eslint will let you:
`const typeArr = results.map(result => result.get('name'));`